File: src/Projects/Android/016-TextGeneratorSystemNetLib/ConsoleClient/src/main/java/org/csystem/app/generator/text/server/Application.java
package org.csystem.app.generator.text.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ConfigurableApplicationContext;

@SpringBootApplication
public class Application {
	public static ConfigurableApplicationContext context;
	public static void main(String[] args)
	{
		context = SpringApplication.run(Application.class, args);
	}
}

File: src/Projects/Android/ChatSystem/Server/src/main/java/org/csystem/app/chat/server/Application.kt
package org.csystem.app.chat.server

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.context.annotation.ComponentScan
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication
@ComponentScan(basePackages = ["org.csystem"])
@EntityScan(basePackages = ["org.csystem"])
@EnableJpaRepositories("org.csystem")
open class Application

fun main(args: Array<String>)
{
    SpringApplication.run(Application::class.java, *args)
}
